dbt_subprojects/dex/models/automated_trades/_schema.yml

```diff
@@ -69,9 +63,9 @@ models:
       - &amount_usd
         name: amount_usd
         description: "USD value of the trade at time of execution. Can be null if we don't have enough data to calculate the value."
-        data_tests:
-          - dbt_utils.accepted_range:
-              max_value: 1000000000 # $1b is an arbitrary number, intended to flag outlier amounts early
+        # data_tests:
+        #  - dbt_utils.accepted_range:
+        #     max_value: 1000000000 # $1b is an arbitrary number, intended to flag outlier amounts early
```

Comment:
was this failing, hence why you commented out? if so, should we be concerned with trades this large?
we run this test in prod on `dex.trades` and helps catch outliers asap

Reply:
just forgot about it and should only be a thing in the mapped model, the unmapped model will throw errors